### ISSUE TITLE
Lack of trailing newline in .d files

### ISSUE THREAD

**Issue by ISSOtm** [Hawkbat/hgbasm#40]
Makefile dependency files lack a trailing newline, making working with them in command-line problematic

```
for line in $(cut -d ":" -f 2 $TMP_DEPFILE); do
    if [ "$line" != "src/maps.asm" ]; then
        echo "$line: ;" >> deps/maps.d
    fi
done
```
Sample output:
```
...
obj/maps.o deps/maps.d: src/maps.asm
obj/maps.o deps/maps.d: src/res/maps/village/village.asm
obj/maps.o deps/maps.d: src/maps/village.asmsrc/res/hardware.inc/hardware.inc: ;
src/constants/enums.asm: ;
src/constants/text.asm: ;
src/constants/maps.asm: ;
...
```

**Comment by Hawkbat** [comment]
Thought I'd fixed this already but it's possible I forgot to push it upstream. I'll look into this tomorrow.

**Comment by Hawkbat** [comment]
@ISSOtm This was fixed in v2.1.1 of hgbasm-cli last month. Can you update and try again?

**Comment by ISSOtm** [comment]
Seems like it's fixed. Thanks!
